### PR TITLE
Dependencies and Renaming

### PR DESCRIPTION
**Pull request by jhazentia**
This PR includes the following changes:
- Update pre-commit dependencies
- Rename radio to sigan or Signal Analyzer
- Change minimum Python to 3.7
- Use pip-tools for managing dependencies
- Use broad range of version numbers in requirements to reduce dependency conflicts
- Use scos-actions 1.0.0
- Change USRPLocation get_lan_long() to get_location(), return height

Propose to tag release as version 0.1.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,20 +56,56 @@ Below are steps to run scos-sensor with the scos-usrp plugin:
...
-```python
+```bash
 python3 -m venv ./venv
 source venv/bin/activate
 python3 -m pip install --upgrade pip # upgrade to pip>=18.1
 python3 -m pip install -r requirements-dev.txt
 ```
 
+The requirements.txt file is intended for dependencies needed for installation into
+scos-sensor. It does not include dependencies already in scos-sensor.
```
Comment (dboulware) [inline README.md:74]: This is not true.

File: scos_usrp/discover/__init__.py [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
 
-actions = {
-    "monitor_usrp": RadioMonitor(radio),
-    "sync_gps": SyncGps(gps),
-}
+actions = {"monitor_usrp": MonitorSignalAnalyzer(sigan), "sync_gps": SyncGps(gps)}
 
 logger.debug("scos_usrp: ACTION_DEFINITIONS_DIR =  " + ACTION_DEFINITIONS_DIR)
-yaml_actions, yaml_test_actions = init(radio=radio, yaml_dir=ACTION_DEFINITIONS_DIR)
+yaml_actions, yaml_test_actions = init(sigan=sigan, yaml_dir=ACTION_DEFINITIONS_DIR)
```
Comment (dboulware) [inline scos_usrp/discover/__init__.py:15]: Just in case this should add gps = gps in the init call

**Review by dboulware — CHANGES_REQUESTED** [review]
Looks good. Just a couple minor comments. See in-line comments.

File: README.md (hunk 1 above)
Comment (jhazentia) [reply README.md]: Fixed. Good catch, thanks!

**Review by dboulware — APPROVED** [review]

**Pull request merged**